test/unit/utils/dom.test.js

```diff
@@ -686,3 +686,48 @@ QUnit.test('isSingleLeftClick() checks return values for mousedown event', funct
 
   assert.ok(Dom.isSingleLeftClick(mouseEvent), 'a touch event on simulated mobiles is a single left click');
 });
+
+QUnit.test('Dom.copyStyleSheetsToWindow() copies all style sheets to a window', function(assert) {
+  const fakeWindow = document.createElement('div');
```

Comment:
This tests would need to use async to wait for the <link> el.

```suggestion
  const fakeWindow = document.createElement('div');
  const done = assert.async();

  assert.expect(7);
```